vs2017/SumatraPDF.vcxproj

```diff
@@ -335,6 +335,7 @@
     <ClCompile Include="..\src\SumatraPDF.cpp" />
     <ClCompile Include="..\src\SumatraStartup.cpp" />
     <ClCompile Include="..\src\Tests.cpp" />
+    <ClCompile Include="..\src\TestTextSearch.cpp" />

```

Comment:
You seem to have manually updated the files. The right way is to change `premake5.lua` and/or `premake5.files.lua` and re-run premake to re-generate project files. See https://www.sumatrapdfreader.org/docs/Our-build-system-3114db56b1a54b9bb026da6b92d36a80.html